integration/setup/setup.go

```diff
@@ -30,10 +29,10 @@ const (
 )
 
 // WrapTestMain setup and teardown e2e testing environment.
-func WrapTestMain(c *client.AzureClientSet, g *framework.Guest, h *framework.Host, m *testing.M) {
+func WrapTestMain(g *framework.Guest, h *framework.Host, m *testing.M) {
```

Comment:
I have no idea what this is about and why it is that way but it is apparently not used for anything. So bye bye. Licky boom boom down.